account_cutoff_base/models/account_cutoff.py

```diff
@@ -353,7 +353,8 @@ def _get_mapping_dict(self):
         """return a dict with:
         key = ID of account,
         value = ID of cutoff_account"""
-        self.ensure_one()
+        for record in self:
+            record.ensure_one()
```

Comment:
this is not needed, `self.ensure_one()` verifies that the current recorset holds a single record, on the model level